src/sentry/api/endpoints/group_notes_details.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+
+from rest_framework import status
+from rest_framework.response import Response
+
+from sentry.api.bases.group import GroupEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.models import Activity
+from sentry.api.endpoints.group_notes import NoteSerializer
```

Comment:
you can move NoteSerializer into our pleasantly named path of sentry/api/serializers/rest_framework